config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
+  get 'oauths/oauth'
+  get 'oauths/callback'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.